cluster/ci/monitoring/mixins/grafana_dashboards/canary.jsonnet

```diff
@@ -0,0 +1,59 @@
+local config =  import '../config.libsonnet';
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local legendConfig = {
+        legend+: {
+            sideWidth: 350
+        },
+    };
+
+dashboard.new(
+        'Release-Informing Jobs Dashboard',
+        time_from='now-2d',
```

Comment:
If we're looking at a two-day moving average, we might as well see the last ~week